airdrop/application/services/airdrop_services.py

```diff
@@ -1,9 +1,28 @@
 from airdrop.infrastructure.repositories.airdrop_repository import AirdropRepository
 from jsonschema import validate, ValidationError
 from http import HTTPStatus
+from common.boto_utils import BotoUtils
+from common.utils import generate_claim_signature
+from airdrop.config import SIGNER_PRIVATE_KEY, SIGNER_PRIVATE_KEY_STORAGE_REGION
 
 
 class AirdropServices:
+
+    def get_signature_for_airdrop_window_id(self, amount, airdrop_id, airdrop_window_id, address):
+        try:
+            boto_client = BotoUtils(
+                region_name=SIGNER_PRIVATE_KEY_STORAGE_REGION)
+            private_key = boto_client.get_parameter_value_from_secrets_manager(
+                secret_name=SIGNER_PRIVATE_KEY)
+
+            # TODO: Get contract address from Network
+            contract_address = ''
```

Comment:
can we pass this as an argument, 
So that we can reususe the same function for different contracts.

Reply:
Fixed in the latest commit